azuredisk-csi-1.31.advisories.yaml

```diff
@@ -20,6 +20,10 @@ advisories:
             componentType: go-module
             componentLocation: /usr/bin/azurediskplugin
             scanner: grype
+      - timestamp: 2025-03-10T16:24:58Z
+        type: fixed
+        data:
+          fixed-version: 1.31.5-r0
```

Comment:
Hmm I would expect automation to publish a fixed event, not us. Do you have a link to the PR that resolved this?

Reply:
@powersj it was fixed upstream: https://github.com/kubernetes-sigs/azuredisk-csi-driver/commit/6bf7fac26bf44e92030c8943c3c16b8a70800d3f made it into 1.31.5 https://github.com/kubernetes-sigs/azuredisk-csi-driver/commits/v1.31.5/ version and we picked it up as well on 1.31.5-r0:

```

╰─$ wolfictl sbom azuredisk-csi-1.31-1.31.5-r0.apk | grep crypto
        📦 golang.org/x/crypto v0.36.0 (go-module)

```

Reply:
our PR with the udpate: https://github.com/wolfi-dev/os/pull/45085/files

Reply:
@dnegreira please bring this up with @philroche I'm EOD'ing shortly, but I would expect automation to add fixed events, not humans.

Reply:
@powersj I spoke briefly with @philroche and this should be remediated after we fix the sbom generation issue, will keep it open to see if it eventually gets fixed